src/machine.rs

```diff
@@ -308,6 +307,7 @@ impl<'mir, 'tcx> Evaluator<'mir, 'tcx> {
                 config.tracked_pointer_tags.clone(),
                 config.tracked_call_ids.clone(),
                 config.tag_raw,
+                local_crates.clone(),
```

Comment:
Hm, I guess copying this is not a big deal but please leave a comment in the `Machine` type field that this data is copied elsewhere and hence must never be mutated.

Reply:
Ah, I knew I was missing some "obvious" way to make this better. :D